x-pack/plugins/apm/server/lib/correlations/latency/get_overall_latency_distribution.ts

```diff
@@ -37,9 +41,7 @@ export function getDistributionAggregation(
   };
 }
 
-export async function getOverallLatencyDistribution(
-  options: CorrelationsOptions
-) {
+export async function getOverallLatencyDistribution(options: Options) {
   const { setup } = options;
```

Comment:
Looks like `setup` is only used for getting the `apmEventClient`. We should extract that out of `setup` (although that can wait)

```ts
    const { apmEventClient } = setup;
```

Reply:
I can create an issue for the extra work that can be made there, there were some other things we talked about but were out of scope of this task.

Reply:
A new issue with the left-overs is very welcome, thanks!